tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -387,12 +383,7 @@ public function testEmptyXXXX()
         $errors = $this->Validator->validate([
             'string' => '',
             'array' => [],
-            'file' => [
-                'name' => 'name',
-                'type' => 'type',
-                'tmp_name' => 'tmp_name',
-                'error' => UPLOAD_ERR_NO_FILE,
-            ],
+            'file' => new UploadedFile('', 0, UPLOAD_ERR_NO_FILE, null, null),
```

Comment:
The file attachment already changed from an array to an `UploadedFile` object.
Refer: https://book.cakephp.org/5/en/appendices/5-0-migration-guide.html#http